src/api/vehicles/index.ts

```diff
@@ -0,0 +1,22 @@
+import Hapi from '@hapi/hapi';
+import { IVehiclesService } from '../../service/interface';
+import { ServiceJourneyVehicleQueryVariables } from '../../service/types';
+import { getVehiclesRequest } from './schema';
+
+export default (server: Hapi.Server) => (service: IVehiclesService) => {
+  server.route({
+    method: 'GET',
+    path: '/bff/v2/vehicles/service-journey',
```

Comment:
Should this be plural? 🙈 

```suggestion
    path: '/bff/v2/vehicles/service-journeys',
```